modules/virt-verify-status-bootsource-update.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-verify-status-bootsource-update_{context}"]
+= Verify status of automatic updating for a boot source
+
+You can verify the status of a boot source to confirm that automatic importing and updating is enabled or disabled.
+
+The `status` section of each boot sources listed in the `dataImportChronTemplates` of the `HyperConverged` CR indicates if automatic importing and updating is enabled or disabled. Examples of statuses include:
+
+* `status:  {}` (null) indicates that automatic importing and updating is neither enabled or disabled. For example, a newly created user-defined boot source will have a status of null.
+* `status: commonTemplate: true` indicates that automatic importing and updating has been enabled for a system-defined (`commonTemplate`) boot source.
+
+.Procedure
+
+. List the `dataImportChronTemplates` in the `HyperConverged` custom resource CR. All user-defined boot sources are displayed.
+. Verify the status of a user-defined boot source by examining the value of the `status` section for the boot source.
```

Comment:
This sentense can be removed, I think. I'm not sure the status tells something about user-defied boot sources (maybe the lack of sub-fields, but I find it confusing).

Currently, there are two fields in the status field. One of them is described above and the other is not documented at all (it's related to the non-documented feature of modifying a system-defined boot source, so there is no point to mention it).

Both of these status fields are only related to system-defined boot sources.

Reply:
I changed that last step to simply: 

_Verify the status of the boot source._

We need to list both steps the user would take, even though they are very simple steps, but I think the status itself is self-documenting as you have it. So trimming down the doc makes sense in this case.